docs/admin-guide/annotations.md

```diff
@@ -35,17 +35,25 @@ The frequency from the [ExAC][exac] population database.
 
 - Key: `EXACAF`
 - Value: `Float`
-- Tools: [VEP][vep], [SnpEff][snpeff], [genmod][genmod], [vcfanno][vcfanno] 
+- Tools: [VEP][vep], [SnpEff][snpeff], [genmod][genmod], [vcfanno][vcfanno]
 
 
 #### ExAC_MAX_AF ####
 
-The maximum allele frequency of all populations [ExAC][exac] population database. 
+The maximum allele frequency of all populations in [ExAC][exac] population database.
 
 - Key: `EXAC_MAX_AF`
 - Value: `Float`
 - Tools: custom made, we have modified the exac file and use [genmod][genmod]
 
+#### GnomAD ####
+
+The maximum allele frequency of all populations in [gnomAD][gnomad] population database.
+
+- Key: `gnomAD_AF`
```

Comment:
Good! If you feel like it, it would sure not hurt making the parse test for this field a bit more explicit as well. Ideally we would like the load/parse tests to be a readable and living set of documentation for the keys (although I am very aware we are not there yet).